themes/frontierline/content-views/content-summary.php

```diff
@@ -1,21 +1,27 @@
 <?php
 /**
- * Display a post with medium image and summary.
+ * Display a post with medium image (if enabled) and summary.
  */
+
+$post_classes = 'post-summary';
+if (get_theme_mod('frontierline_post_thumbnail') !== '1') {
+  $post_classes .= ' post-thumb-disabled';
+}
 ?>
 
-<article id="post-<?php the_ID(); ?>" <?php post_class('post-summary'); ?>>
+<article id="post-<?php the_ID(); ?>" <?php post_class($post_classes); ?> >
```

Comment:
I am not sure if this will work correctly. [`post_class`](https://codex.wordpress.org/Function_Reference/post_class) should be probably called with an array in case there are multiple classes.